pytest_catchlog.py

```diff
@@ -122,6 +134,15 @@ def __init__(self, config):
         self.formatter = logging.Formatter(
                 get_option_ini(config, 'log_format'),
                 get_option_ini(config, 'log_date_format'))
+        terminal = py.io.TerminalWriter(sys.stderr)  # pylint: disable=no-member
```

Comment:
BTW, what is the rationale of wrapping `stderr` with `TerminalWriter`? AFAIK, `logging.StreamHandler` works well with `stderr` directly.